ERCS/draft_encrypted-types.md

```diff
@@ -0,0 +1,596 @@
+---
+eip: xxxx
```

Comment:
```suggestion
eip: 7995
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.